### ISSUE TITLE
Main panics when Scraper used in `while` or `loop`

### ISSUE THREAD

**Issue by ghost** [Gonzih/crabler#5]
When scraper is used within a `while` or `loop` it throws the following error:
`thread 'main' panicked at 'Could not start logging: SetLoggerError(())'` originating from [femme-2.1.1/src/pretty.rs:16:35](https://github.com/lrlna/femme/blob/master/src/pretty.rs)

**Comment by Gonzih** [comment]
Code sample? Have no idea what you are trying to achieve, but this scraper is not intended to be used in a loop. You do recursion by opening links if needed.

**Comment by ghost** [comment]
Using the crabler example, my code that's throwing errors looks similar to this:
```
#[async_std::main]
async fn main() -> Result<()> {
    loop {
        let scraper = Scraper {};

        // Run scraper starting from given url and using 20 worker threads
        scraper.run(Opts::new().with_urls(vec!["https://www.rust-lang.or/"]).with_threads(20)).await?;
        task::sleep(Duration::from_secs(3600)).await;
     }
```
The intention is to get data from a site every hour while the code is running. If the scraper is not intended to be used that way then that's fair enough and this issue can be closed.

**Comment by Gonzih** [comment]
I published version 0.1.22 that disables logging initialization if `debug` flag is not present, I think this should fix your problem.
Please let me know if it works.

**Comment by ghost** [comment]
Yes, this solves my issue! Thank you very much :)
